### DISCUSSION TITLE
Question about last versions

### DISCUSSION THREAD

**Discussion by mariofix** [aminalaee/sqladmin#499] [General]
Hi! I was so happy to see the custom actions support added, but since it's not on pypi yet, is there a way to try them (short of cloning the repo in my project), I've tried adding it to my pyproject.toml file as 
sqladmin = {git = "https://github.com/aminalaee/sqladmin.git", rev = "main"}

but poetry/pip complains with python 3.10 and 3.11
```
Updating dependencies
Resolving dependencies... (3.9s)

Unable to determine package info for path: /home/mariofix/.cache/pypoetry/virtualenvs/project-t7_AZ976-py3.11/src/sqladmin
``` 

How everybody else trying this feature?

**Comment by aminalaee** [comment]
Hey, 
I will try to do the release today, as there's no new PRs to be included.
And about your question, TBH I don't know why this is happening.

**Comment by aminalaee** [comment]
Just released latest version: https://github.com/aminalaee/sqladmin/releases/tag/0.11.0

**Comment by anton-petrov** [comment]
Use commit instead of branch name. For example:
```
sqladmin = {git = "https://github.com/aminalaee/sqladmin.git", rev = "b38324a"}
```
